jekyll/_cci2/contexts.md

```diff
@@ -40,9 +42,19 @@ workflows:
             context: org-global
 ```
 
-To rerun a job and use the context, it must be rerun from the Workflows page of the CircleCI application. Jobs invoked using the Rebuild button on the Builds page will **not** use the context defined in the workflow. 
+To rerun a job and use the context, it must be rerun from the Workflows page of the CircleCI application. 
```

Comment:
might want to highlight `must` here, this is something which trips people up all the time

Reply:
thanks, added